src/Symfony/Component/Uid/Command/InspectUlidCommand.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Uid\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Uid\Ulid;
+
+class InspectUlidCommand extends Command
+{
+    protected static $defaultName = 'ulid:inspect';
```

Comment:
what about a single `uid:inspect` command that could deal with ULID and UUID?

Reply:
I would prefer to have separate commands, otherwise we need an extra argument to specify if the inspected uid is supposed to be an ulid or an uuid. We cannot guess it.

Reply:
actually, we can: `$isUlid = \strlen($uid) === 26` if you want the strict definition. And if you use `Ulid::fromString()`, there is no point distinguishing as you can pass a uuid to `Ulid::fromString` and it will be converted (both Ulid and Uuid are different representations of a 128bits number).

Reply:
There is a difference on the timestamp part because of different epochs. If I want to inspect `3a2be45c-5a62-11eb-9956-537bd3b4c9e7`, we cannot know if I want to interpret it as an ulid or as an uuidv1.